src/main/java/com/custom/fuelsdk/ETClickEvent.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.custom.fuelsdk;
+
+import com.custom.fuelsdk.internal.ClickEvent;
+import com.exacttarget.fuelsdk.ETSoapObject;
+import com.exacttarget.fuelsdk.annotations.ExternalName;
+import com.exacttarget.fuelsdk.annotations.InternalName;
+import com.exacttarget.fuelsdk.annotations.SoapObject;
+import com.exacttarget.fuelsdk.internal.EventType;
+
+import java.util.Date;
+
+/**
+ * To work around the bug in FuelSDK the class is extended and modified.
+ * Refer to the below URL
+ * https://github.com/salesforce-marketingcloud/FuelSDK-Java/pull/98
+ */
+@SoapObject(internalType = ClickEvent.class)
+public class ETClickEvent extends ETSoapObject {
+
+  @InternalName("objectID")
+  private String id;
+
+  @ExternalName("sendID")
+  private Integer sendID;
+
+  @ExternalName("subscriberKey")
+  private String subscriberKey;
+
+  @ExternalName("eventDate")
+  private Date eventDate;
+
+  @ExternalName("eventType")
+  private EventType eventType;
+
+  @ExternalName("triggeredSendDefinitionObjectID")
+  private String triggeredSendDefinitionObjectID;
+
+  @ExternalName("batchID")
+  private Integer batchID;
+
+  @ExternalName("urlid")
+  private Integer urlid;
+
+  @ExternalName("url")
+  private String url;
+
+  /**
+   * @return the id
```

Comment:
I don't think the javadoc is really useful for simple getters and setters.

Reply:
The original ETClickEvent from FuelSDK was copied to fix a bug with methods getURLID.   These comments were part of the original code.